tools/CreateRuleFabricBot/CreateRuleFabricBot/Rules/PullRequestLabel/PullRequestLabelFolderCapability.cs

```diff
@@ -131,11 +134,33 @@ private string ToConfigString(CodeOwnerEntry entry)
             return result;
         }
 
+        private string GetFileContents(string fileOrUri)
+        {
+            if (File.Exists(fileOrUri))
+            {
+                return File.ReadAllText(fileOrUri);
+            }
+
+            // try to parse it as an Uri
+            Uri uri = new Uri(fileOrUri, UriKind.Absolute);
+            if (uri.Scheme.ToLowerInvariant() !="https")
+            {
+                throw new ArgumentException("Cannot download off non-https uris");
+            }
+
+            // try to download it.
+            using (HttpClient client = new HttpClient())
+            {
+                HttpResponseMessage response = client.GetAsync(fileOrUri).ConfigureAwait(false).GetAwaiter().GetResult();
+                return response.Content.ReadAsStringAsync().ConfigureAwait(false).GetAwaiter().GetResult();
+            }
+        }
+
         private List<CodeOwnerEntry> ReadOwnersFromFile()
         {
             List<CodeOwnerEntry> entries = new List<CodeOwnerEntry>();
             string line;
-            using (StreamReader sr = new StreamReader(_codeownersFile))
+            using (StringReader sr = new StringReader(GetFileContents(_codeownersFile)))
```

Comment:
Where is the multiple labels coming into play?

Reply:
From the empty string in the exclude entry in the payload. This is based on how the service works.